packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -58,20 +58,20 @@ export function isHostClientMobile(): boolean {
 /**
  * @hidden
  * Helper function which indicates if current API is supported on mobile or not.
- * @returns SdkError if host client is not android/ios or if the requiredVersion is not
+ * @throws SdkError if host client is not android/ios or if the requiredVersion is not
  *          supported by platform or not. Null is returned in case of success.
  *
  * @internal
  */
-export function isApiSupportedOnMobile(requiredVersion: string = defaultSDKVersionForCompatCheck): SdkError {
+export function throwExceptionIfApiIsNotSupported(requiredVersion: string = defaultSDKVersionForCompatCheck): void {
   if (!isHostClientMobile()) {
     const notSupportedError: SdkError = { errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM };
-    return notSupportedError;
-  } else if (!isAPISupportedByPlatform(requiredVersion)) {
+    throw notSupportedError;
+  } else if (!isCurrentSDKVersionAtLeast(requiredVersion)) {
     const oldPlatformError: SdkError = { errorCode: ErrorCode.OLD_PLATFORM };
-    return oldPlatformError;
+    throw oldPlatformError;
   }
-  return null;
+  return undefined;
```

Comment:
I think you can just delete this return statement entirely. Since it's a void function there's no need